content/help/ops/setup/injection/index.md

```diff
@@ -0,0 +1,242 @@
+default        Active    18d       enabled
+istio-system   Active    3d
+kube-public    Active    18d
+kube-system    Active    18d
+{{< /text >}}
+
+The `namespaceSelector` for opt-out will look like the following:
+
+{{< text bash >}}
+$ kubectl get mutatingwebhookconfiguration istio-sidecar-injector -o yaml | grep "namespaceSelector:" -A5
+  namespaceSelector:
+    matchExpressions:
+    - key: istio-injection
+      operator: NotIn
+      values:
+      - disabled
+  rules:
+  - apiGroups:
+    - ""
+{{< /text >}}
+
+The injection webhook will be invoked for pods created in namespaces
+without the `istio-injection=disabled` label.
+
+{{< text bash >}}
+$ kubectl get namespace -L istio-injection
+NAME           STATUS    AGE       ISTIO-INJECTION
+default        Active    18d
+istio-system   Active    3d        disabled
+kube-public    Active    18d       disabled
+kube-system    Active    18d       disabled
+{{< /text >}}
+
+Verify the application pod's namespace is labeled properly and (re) label accordingly, e.g.
+
+{{< text bash >}}
+$ kubectl label namespace istio-system istio-injection=disabled --overwrite
+{{< /text >}}
+
+(repeat for all namespaces in which the injection webhook should be invoked for new pods)
+
+{{< text bash >}}
+$ kubectl label namespace default istio-injection=enabled --overwrite
+{{< /text >}}
+
+1. Check default policy
+
+Check the default injection policy in the `istio-sidecar-injector` `configmap`.
+
+{{< text bash >}}
+$ kubectl -n istio-system get configmap istio-sidecar-injector -o jsonpath='{.data.config}' | head
+policy: enabled
+template: |-
+  initContainers:
+  - name: istio-init
+    image: "docker.io/jasonayoung/proxy_init:d49fa0a7f7d17f25552ad749d23f8ac73596e0cc"
+    args:
+    - "-p"
+    - [[ .MeshConfig.ProxyListenPort ]]
+    - "-u"
+    - 1337
+{{< /text >}}
+
+Allowed policy values are `disabled` and `enabled`. The default policy
+only applies if the webhook’s `namespaceSelector` matches the target
+namespace. Unrecognized policy values default to `disabled`.
+
+1. Check the per-pod override annotation
+
+The default policy can be overwritten with the
+`sidecar.istio.io/inject` annotation in the _pod template spec’s
+metadata_. The deployment’s metadata is ignored. Annotation value of
+`true` forces the sidecar to be injected while a value of `false`
+forces the sidecar to _not_ be injected.
+
+The following annotation overrides whatever the default `policy` was
+to force the sidecar to be injected.
+
+{{< text bash >}}
+$ kubectl get deployment sleep -o yaml | grep "sidecar.istio.io/inject:" -C3
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true"
+      labels:
+        app: sleep
+{{< /text >}}
+
+## Pods cannot be created at all
+
+Run ```kubectl describe -n namespace deployment name``` on the failing
+pod's deployment. Failure to invoke the injection webhook will
+typically will be captured in the event log.
+
+### `x509 certificate related errors`
```

Comment:
I think simply:

x509 certificate errors

is sufficient, no need for backticks either.